.vsts-pipelines/builds/ci-public.yml

```diff
@@ -0,0 +1,40 @@
+trigger:
```

Comment:
Can these two files import a common template as well somehow? Just wondering if we can further reduce duplication by putting the "NodeTool" custom step stuff put in a single file.

Reply:
That's something we can experiment with later, once we've seen a few more special cases. I'm not sure we know enough about the ways things will be different between repos to be useful yet.